### DISCUSSION TITLE
'Some variables become negative' when I simulate oral formulation models

### DISCUSSION THREAD

**Discussion by wangwei1619** [Open-Systems-Pharmacology/Forum#804] [Questions & Problems] [answered]
Hi, all

When I simulate some oral formualtion models, a notice appears like this:
![image](https://user-images.githubusercontent.com/54141158/67615665-d85add00-f801-11e9-942e-8c5c534b5ea1.png)
I attach the PK-Sim file to this issue. In this file, the simulation 'iv' and 'suspension' are fitted to observed data, and the fitted parameters are used to consturct models of other formulations. Among these formulations, 'reference', 'T1' and 'T2' would result in the above notice (with different specifi  variables) .
How can I resolve this problem? 

[Lacidipine.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/3774555/Lacidipine.zip)

**Comment by Yuri05** [comment, marked as answer]
For all oral formulations - **except for _Particle dissolution_** -  (InVitro) dissolved fraction of dose is assumed **monotonically increasing**.
In all your "problematic" formulations ('reference', 'T1' and 'T2') it's not the case. E.g.:
![grafik](https://user-images.githubusercontent.com/25061876/67618723-772c0d00-f7f3-11e9-949a-12639af620fa.png)
This is not allowed and leads to negative concentrations in the lumen.
![grafik](https://user-images.githubusercontent.com/25061876/67618744-b9ede500-f7f3-11e9-8aa3-09f305534008.png)
Please make sure that fraction (dose) in all table formulations is non decreasing.

@AndreDlm @KatrinCoboeken FYI

**Comment by wangwei1619** [comment]
Hi, Yuri05
Thanks for your help so much.  But dissolved drug sometimes would get precipitated in dissolution test, which makes the concentration decrease. Is there any way to handle this trouble?
By the way, I am curious about how you can run the simulation just like the figure you post. Because PK-Sim just gives out the error notice and stop the simulation when I run it. Would you mind tell me how to make that and how to inspect this sort of problems? Thanks.
